recipes/sdl/all/conanfile.py

```diff
@@ -127,23 +129,20 @@ def requirements(self):
                 self.requires("pulseaudio/14.2")
             if self.options.opengl:
                 self.requires("opengl/system")
-
-    def _check_pkg_config(self, option, package_name):
-        if option:
-            pkg_config = tools.PkgConfig(package_name)
-            if not pkg_config.provides:
-                raise ConanInvalidConfiguration("package %s is not available" % package_name)
+            if self.options.nas:
+                self.requires("nas/1.9.4")
+            if self.options.jack:
+                raise ConanInvalidConfiguration("Package for 'jack' is not available (yet)")
+            if self.options.esd:
+                raise ConanInvalidConfiguration("Package for 'esd' is not available (yet)")
+            if self.options.wayland:
+                self.requires("wayland/1.19.0")
```

Comment:
```suggestion
                self.requires("wayland/1.19.0")
                self.requires("xkbcommon/1.3.0")
                self.requires("egl/system")
```
Not tested, but it should solve 
```
-- Checking for modules 'wayland-client;wayland-scanner;wayland-egl;wayland-cursor;egl;xkbcommon'
--   Package 'egl', required by 'virtual:world', not found
--   Package 'xkbcommon', required by 'virtual:world', not found
```

Reply:
👍  I'll work on the `nas` subgraph first. I'm afraid on the `wayland` branch we will find the same issue with the transitive deps.

Reply:
@ericLemanissier `wayland` should work now. It required a patch to use `wayland-scanner` from the _build_ context (although this recipe will fail in cross-building scenario, but better to be prepared).

Anyway, it requires this PR to be merged first: https://github.com/conan-io/conan-center-index/pull/6932